packages/domain-picker/test/search-and-select-domain-suggestion.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { DataStatus, DomainSuggestion } from '@automattic/data-stores/src/domain-suggestions';
+
+/**
+ * Internal dependencies
+ */
+import '../matchMedia.mock';
+import DomainPicker from '../src/domain-picker';
+import { useDomainSuggestions } from '../src/hooks/use-domain-suggestions';
+
+// TODO: should we extract to separate mock data file/folder?
```

Comment:
I think so. We could use a `__mocks__` folder to mock the different hooks/modules (https://jestjs.io/docs/manual-mocks).

Reply:
Agreed — this will be worked on in a separate issue/PR